spec/02-integration/05-proxy/01-router_spec.lua

```diff
@@ -77,9 +77,15 @@ describe("Router", function()
         {
           name = "api-4",
           upstream_url = "http://httpbin.org/basic-auth",
-          uris = { "/user" },
+          uris = { "/private" },
```

Comment:
`/user` conflicts with the new `"[uri] with a regex"` test below which requests `/users/foo/profile`. And since `/private` is a better alias for the upstream_uri associated with this test (`/basic-auth`), story checks out.